### PR TITLE
[init_cfg.json.j2]: only enable gbsyncd feature for vs platform

### PR DESCRIPTION
**Pull request by lguohan**
currently only vs platform has gdbsyncd feature built

Signed-off-by: Guohan Lu <lguohan@gmail.com>

**- Why I did it**
currently only vs platform has gdbsyncd feature built

**- How I did it**
use jinja2 template to enable the feature only on vs platform.

**- How to verify it**
will download the image built and try.

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by jleveque — APPROVED** [review]

File: files/build_templates/init_cfg.json.j2 [hunk 1]
```diff
@@ -26,7 +26,9 @@
                    ("snmp", "enabled", true, "enabled"),
                    ("swss", "enabled", false, "enabled"),
                    ("syncd", "enabled", false, "enabled"),
+{% if sonic_asic_platform == "vs" %}
```
Comment (prsunny) [inline files/build_templates/init_cfg.json.j2:29]: Just checking, should we have it as "include_gbsyncd" and add as a compilation option?

Comment (lguohan) [reply files/build_templates/init_cfg.json.j2:29]: in the future yes, not right now. gbsyncd currently is hardcoded to vs platform.

Comment (lguohan) [reply files/build_templates/init_cfg.json.j2:29]: btw, in the future, the gbsyncd should be by default embedded into the the image and loaded at the runtime based on the platform.

Comment (prsunny) [reply files/build_templates/init_cfg.json.j2:29]: ok

**Review by prsunny — DISMISSED** [review]

**Review by jleveque — DISMISSED** [review]

**Comment by lguohan** [comment]
retest vsimage  pleass

**Comment by lguohan** [comment]
retest vsimage please

**Review by jleveque — APPROVED** [review]

**Pull request merged**